File: src/main/java/technology/touchmars/outside/feign/wechat/config/WechatMiniProgramProperties.java
/**
 * 
 */
package technology.touchmars.outside.feign.wechat.config;

import org.springframework.boot.context.properties.ConfigurationProperties;

import lombok.Data;

/**
 * @author jhcao
 *
 */
@Data
@ConfigurationProperties(prefix = "wechat.mini")
public class WechatMiniProgramProperties {
	private String clientId;
	private String secret;
}

File: src/main/kotlin/technology/touchmars/outside/feign/wechat/config/WechatMiniProgramProperties.kt
/**
 *
 */
package technology.touchmars.outside.feign.wechat.config

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * @author jhcao
 */
@ConfigurationProperties(prefix = "wechat.mini")
class WechatMiniProgramProperties {
    val clientId: String? = null
    val secret: String? = null
}
